src/main/groovy/edu/oregonstate/mist/personsapi/core/JobObject.groovy

```diff
@@ -4,25 +4,104 @@ import com.fasterxml.jackson.annotation.JsonFormat
 import com.fasterxml.jackson.annotation.JsonIgnore
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties
 import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
 import edu.oregonstate.mist.api.jsonapi.ResultObject
 import groovy.transform.InheritConstructors
 
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+
 @JsonIgnoreProperties(ignoreUnknown=true) //when deserializing, ignore unknown fields
 class JobObject {
+    private static ObjectMapper mapper = new ObjectMapper().registerModule(new JavaTimeModule())
+
+    private static final String accruesLeaveTrue = 'Y'
+    private static final String accruesLeaveFalse = 'N'
+
+    public static final String activeJobStatus = "Active"
+    // Hard code job statuses and those descriptions here since there is no validation table
+    private static final def jobStatusDict = [
+            'A': activeJobStatus,
```

Comment:
I guess I just wanted to keep code like this cleaner: https://github.com/osu-mist/persons-api/blob/802df731ff6b0d30ae379424dbe42e59f48cc531/src/main/groovy/edu/oregonstate/mist/personsapi/PersonsResource.groovy#L273

`jobStatusDict` is meant to be private, since `'A'` is a database specific status. I wanted to abstract the backend db terminology from the personsresource.

Reply:
Hmm, I guess I could do `public static final String activeJobStatus = jobStatusDict['A']`. I'll try that.

Reply:
Doing `public static final String activeJobStatus = jobStatusDict['A']` compiled, but threw an error when running the tests due to 
```
java.lang.NoClassDefFoundError: Could not initialize class edu.oregonstate.mist.personsapi.core.JobObject
```